beekeeper-metadata-cleanup/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beekeeper-parent</artifactId>
+    <groupId>com.expediagroup</groupId>
+    <version>2.1.1-SNAPSHOT</version>
```

Comment:
might want to check with @vedantchokshi before merging because I think the other PR bumps this to 3.0.0-SNAPSHOT

Reply:
yeah I was going to wait till his stuff was in before I add anything 👌🏻